components/button/button.css

```diff
@@ -0,0 +1,114 @@
+/* Button CSS */
+.btn {
```

Comment:
- can also add white-space: nowrap 
  because generally, we don't want buttons text to go next line.
- can use position: relative, so that if you add
  icons in future, they will be relative to this class.

Reply:
Applied your suggestions,
also created buttons with icons
![image](https://user-images.githubusercontent.com/65483354/151779184-83f61a7c-59e0-4a4c-93b1-c48d2a7d2441.png)